packages/shared/src/hooks/useMutateFilters.ts

```diff
@@ -163,17 +165,54 @@ export default function useMutateFilters(
   const queryClient = useQueryClient();
   const shouldFilterLocally = shouldFilterLocallyProp || !user;
 
+  const { mutateAsync: removeFiltersRemote } = useMutation<
+    unknown,
+    unknown,
+    SourceMutationProps & { key?: string },
+    () => void
+  >(
+    ({ source, key = 'excludeSources' }) =>
+      gqlClient.request(REMOVE_FILTERS_FROM_FEED_MUTATION, {
+        filters: {
+          [key]: [source.id],
+        },
+      }),
+    {
+      // onMutate: onUnblockSource,
```

Comment:
This is missed?

Reply:
This is the generic function I've written for refactoring and do not repeat the mutation over follow/notify/etc...
I'm removing the comment! :)